### DISCUSSION TITLE
how to change the language?

### DISCUSSION THREAD

**Discussion by Fanvarli** [Rem0o/FanControl.Releases#1531] [General]
Salam!
There are localization files in the folder, I took them and replaced them with a similar file, but the language did not change. How to change? What are Microsoft.Win32.TaskScheduler.resources.dll files?

**Comment by Rem0o** [comment]
Those are the localization resources for the task scheduler dll I'm using. FanControl itself doesn't have multi-language support.

**Comment by Fanvarli** [comment]
xuevo konecno, but ok...
